### ISSUE TITLE
Multiple Runs (Parallel Execution)

### ISSUE THREAD

**Issue by triptithawait** [triptithawait/skills-copilot-codespaces-vscode#6]

**Comment by triptithawait** [comment]
import threading

def train_model(run_id):
    model = build_model()
    print(f"Training Run {run_id}")
    model.fit(X_train, y_train, epochs=5, batch_size=128, verbose=0)
    loss, acc = model.evaluate(X_test, y_test, verbose=0)
    print(f"Run {run_id} - Accuracy: {acc:.4f}")

threads = []
for i in range(3):  # 3 parallel runs
    t = threading.Thread(target=train_model, args=(i+1,))
    threads.append(t)
    t.start()

for t in threads:
    t.join()
